### DISCUSSION TITLE
What is the whole words for txn (in Transaction)？

### DISCUSSION THREAD

**Discussion by littletreezlx** [isar/isar#1438] [Help]
I'm really confused...

**Comment by nobkd** [comment]
`txn` is an abbreviation for the word `transaction`.

Or what is your question?

**Reply by littletreezlx** [reply]
sry, I just have not heard the abbr 'txn', also not found on wiki. So I'm confused when using this method, thinking about what's the three words in it.

Just got it somewhere. [TXN] means the transaction number assigned to every transaction in the ISSUING BANK online system.

**Reply by nobkd** [reply]
When I search for `abrreviation txn` i get the following immediately :shrug: 
https://www.abbreviations.com/TXN
https://www.allacronyms.com/TXN
